yarn.lock

```diff
@@ -15699,6 +15699,36 @@ webpack-sources@^2.1.1:
     source-list-map "^2.0.1"
     source-map "^0.6.1"
 
+webpack@^5.11.1:
+  version "5.11.1"
+  resolved "https://registry.yarnpkg.com/webpack/-/webpack-5.11.1.tgz#39b2b9daeb5c6c620e03b7556ec674eaed4016b4"
+  integrity sha512-tNUIdAmYJv+nupRs/U/gqmADm6fgrf5xE+rSlSsf2PgsGO7j2WG7ccU6AWNlOJlHFl+HnmXlBmHIkiLf+XA9mQ==
+  dependencies:
+    "@types/eslint-scope" "^3.7.0"
+    "@types/estree" "^0.0.45"
+    "@webassemblyjs/ast" "1.9.1"
+    "@webassemblyjs/helper-module-context" "1.9.1"
+    "@webassemblyjs/wasm-edit" "1.9.1"
+    "@webassemblyjs/wasm-parser" "1.9.1"
+    acorn "^8.0.4"
+    browserslist "^4.14.5"
+    chrome-trace-event "^1.0.2"
+    enhanced-resolve "^5.3.1"
+    eslint-scope "^5.1.1"
+    events "^3.2.0"
+    glob-to-regexp "^0.4.1"
+    graceful-fs "^4.2.4"
+    json-parse-better-errors "^1.0.2"
+    loader-runner "^4.1.0"
+    mime-types "^2.1.27"
+    neo-async "^2.6.2"
+    pkg-dir "^5.0.0"
+    schema-utils "^3.0.0"
+    tapable "^2.1.1"
+    terser-webpack-plugin "^5.0.3"
+    watchpack "^2.0.0"
+    webpack-sources "^2.1.1"
+
```

Comment:
I gather we can get rid of the duplication reported by `yarn why` by updating webpack dep in the bundler too:

```
=> Found "webpack@5.11.0"
info Has been hoisted to "webpack"
info Reasons this module exists
   - "workspace-aggregator-63f55e4a-73ca-4134-8d4a-bb0252cc0639" depends on it
   - Specified in "devDependencies"
   - Hoisted from "_project_#@liferay#npm-bundler#webpack"
   - Hoisted from "_project_#webpack"
   - Hoisted from "_project_#@liferay#js-toolkit-core#webpack"
info Disk size without dependencies: "5.97MB"
info Disk size with unique dependencies: "10MB"
info Disk size with transitive dependencies: "22.53MB"
info Number of shared dependencies: 52
=> Found "@liferay/npm-scripts#webpack@5.11.1"
info This module exists because "_project_#@liferay#npm-scripts" depends on it.
info Disk size without dependencies: "4.84MB"
info Disk size with unique dependencies: "8.88MB"
info Disk size with transitive dependencies: "21.41MB"
info Number of shared dependencies: 52
```

Reply:
Yes, I [got that one](https://github.com/liferay/liferay-frontend-projects/pull/376#issuecomment-752504567) but first I tried to fix the `tsc` issue.